Frontend/src/components/institutionOverview/InstitutionCard/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import SettingsIcon from '@mui/icons-material/Settings';
+import Button from '@mui/material/Button';
+import ListCard from 'components/general/ListCard';
+import InstitutionLeaveButton from 'components/institutionOverview/InstitutionLeaveButton';
+import styles from './institutionCard.module.css';
+
+import getUser from 'shared/services/mock/user';
+
+function InstitutionCard({ institution, onLeft }) {
+  const [user, setUser] = useState({});
+  useEffect(() => {
```

Comment:
It is OK for now but when my branch is merged you can do: `const [user] = useAuth()` and simply `user.name` or any other attribute. So basically, we will not need to make a request to backend.